flakeheaven/_version.py

```diff
@@ -1 +1,9 @@
-__version__ = '0.11.0'
+try:
+    # built-in
+    from importlib import metadata
+except ImportError:
+    # Running on pre-3.8 Python; use importlib-metadata package
+    # external
+    import importlib_metadata as metadata  # type: ignore
+
+__version__ = metadata.version('flakeheaven')
```

Comment:
poetry requires that we define the version in `pyproject.toml` instead of in the code [ref](https://python-poetry.org/docs/pyproject/#version)

this seemed like an acceptable way to avoid having to update the version in two places

Reply:
1 Either this or a bump helper to keep them synced. (Eg commitizen)

2 Does this work on all versions? I seem to recall a `trry/except` `importlib.metadata` depending on the version... (Maybe it's not in the tests?)

3 does this add considerable time to the `import flakeheaven` statement?

Reply:
it seems this does not add considerable time to the import.
using current implementation:
```
python3 -m timeit 'import flakeheaven'                                                                                                                                       
>> 5000000 loops, best of 5: 65 nsec per loop
```
using string version (`__version__ = "0.11.0"`)
```
python3 -m timeit 'import flakeheaven'                                                                                                                                       
>> 5000000 loops, best of 5: 64.8 nsec per loop
```

let me know if you'd like me to test it in a different way